src/component/Rockets.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets, reserveRocket } from '../features/rockets/rocketsSlice';
+import styles from '../styles/Rockets.module.css';
+
+function Rockets() {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const status = useSelector((state) => state.rockets.status);
+  const error = useSelector((state) => state.rockets.error);
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
```

Comment:
- [ ] Kindly also move this code to `App.js` for the same problem as `Missions`.

Reply:
Thanks for the review 👍

Reply:
- Thanks for the review @gemmen29